### DISCUSSION TITLE
Depth of input images (8bit, 12bit, etc)

### DISCUSSION THREAD

**Discussion by ajensen1234** [CERN/TIGRE#538] [Q&A] [answered]
The CB detector that I'm using captures 12-bit images, so the values are between [0,4095]. Multiple different types of reconstruction seem to be giving the same grey swoops and swirls, and I just wanted to make sure that this is not having to do with the rescaling of the images.

I notice that in some of the demos, the head phantom projections have a maximum value in the ~80s, which is obviously not a 12 bit image.

Thanks to anyone who might have an answer!

**Comment by AnderBiguri** [comment, marked as answer]
hi @ajensen1234 ,
range in CT is arbitrary, as its dependent on the hardware output (the ADC), not really physical units. So any range works in TIGRE, within reasonable limits. The only limitation is that the data must be converted into a `single` (`float32`) data format before being used in the recons. This means that the "grey swoops and swirls" (unsure what you mean, do show an image) are unrelated to data type and scale. 

I have myself used 16 bit detectors. 
Now, if you are using raw data from detectors, do remember to log-transform it!

**Reply by ajensen1234** [reply]
Great - that makes perfect sense. I guess I'll need to dig a bit deeper then. I'll put any other questions into different discussion posts so that other people can benefit from them!
